<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImageToMed</class>
 <widget class="QWidget" name="DlgImageToMed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lblImageFile">
             <property name="text">
              <string>Fichier Image:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMedFile">
             <property name="text">
              <string>Fichier MED:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="txtImageFile"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtMedFile"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btnImageFileChooser">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../../resources/XMED_load.png</normaloff>../../../../resources/XMED_load.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMedFileChooser">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../../../resources/XMED_load.png</normaloff>../../../../resources/XMED_load.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAutoLoad">
         <property name="text">
          <string>Charger automatiquement</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
